Sources/Shared/API/Responses/Logbook.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import ObjectMapper
+
+public class LogbookResponse: Mappable {
```

Comment:
although you followed much of the codebase here, i think you'd be better served by making this conform to `ImmutableMappable` (see e.g. `MobileAppConfigAction`) which allows you to drop all of the not-actually-optional optionals in this class

Reply:
I made it conform to `ImmutableMappable `.

However, when doing so I was getting compile errors because there were two identical `request` helper methods, one of them expecting `BaseMappable` and another `ImmutableMappable` (which inherits from `BaseMappable`) so compiler was not sure which one to use.
So I removed the more constraining one for `ImmutableMappable` in 3f76f21fa19fe9bdd9f5381b5b9fd23a1e7d8781